argoverse/evaluation/eval_detection.py

```diff
@@ -0,0 +1,288 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+import argparse
+import logging
+import os
+from collections import defaultdict
+from dataclasses import dataclass, field
+from multiprocessing import Pool
+from pathlib import Path
+from typing import DefaultDict, List, Tuple
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+from argoverse.data_loading.object_classes import OBJ_CLASS_MAPPING_DICT
+from argoverse.data_loading.object_label_record import read_label
+from argoverse.evaluation.detection_utils import (
+    DistFnType,
+    SimFnType,
+    compute_match_matrix,
+    dist_fn,
+    filter_instances,
+    get_ranks,
+    interp,
+)
+
+matplotlib.use("Agg")
```

Comment:
thanks for adding this! I think it had to be added before we import pyplot though, can we move it up to before line 12?